tests/test_connectivity.py

```diff
@@ -382,7 +403,7 @@ def test_compatibility():
     # Test BaseConnectivity
     assert not baseCon.is_compatible(label_con)
```

Comment:
```suggestion
    assert not con_base.is_compatible(label_con)
```

The python convention is to use snake case 🐍.